src/test/store/markers.test.js

```diff
@@ -370,6 +381,189 @@ describe('Marker schema filtering', function() {
   });
 
   it('filters for getMarkerChartMarkerIndexes', function() {
+    const { getMarkerNames } = setup(getProfileForMarkerSchema());
+    expect(
+      getMarkerNames(selectedThreadSelectors.getMarkerChartMarkerIndexes)
+    ).toEqual([
+      'no payload',
+      'payload no schema',
+      'RefreshDriverTick',
+      // Notice: no network markers!
+      'UserTiming',
+      'RandomTracingMarker',
+    ]);
+  });
+
+  it('filters for MarkerChartMarkerIndexes also for upgraded profiles', async function() {
```

Comment:
nit: would be good to mention about the profile version in the test name too (or mentioning that which upgrader is responsible for this).